lib/QubitCsvTransform.class.php

```diff
@@ -24,6 +24,7 @@ class QubitCsvTransform extends QubitFlatfileImport
     public $rowsPerFile = 1000;
     public $preserveOrder = false;
     public $levelsOfDescription;
+    public $sortOrderLogic;
```

Comment:
Suggestion: Maybe `$sortCallback` or `$sortFunction` to make it more explicit that a callable function or method  is expected?

Reply:
Good thinkin'.